mods/mc_worldmanager/commands.lua

```diff
@@ -161,4 +161,18 @@ minetest.register_chatcommand("realmSetSpawn", {
 
         return true, "Updated spawnpoint for realm with ID: " .. param
     end,
-})
\ No newline at end of file
+})
+
+minetest.register_chatcommand("realmCleanup", {
+    privs = {
+        interact = true,
```

Comment:
Probably want to consider setting this to `{teacher = true}` instead of `interact`.

Reply:
Good catch; I'll update those right now and push again

Reply:
Updated!